docs/tempo/website/configuration/gcs.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Google Cloud Storage (GCS)
+---
+
+# GCS Configuration
+GCS backend is configured in the storage block.  Tempo requires a dedicated bucket since it maintains a top-level object structure and does not support a custom prefix to nest within a shared bucket.
```

Comment:
```suggestion
GCS backend is configured in the storage block. Tempo requires a dedicated bucket since it maintains a top-level object structure and does not support a custom prefix to nest within a shared bucket.
```